server.js

```diff
@@ -1,33 +1,68 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
+import express from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+import data from "./data/netflix-titles.json";
+
+// Defines the port the app will run on. Defaults to 8080, but can be
 // overridden when starting the server. For example:
 //
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+const port = process.env.PORT || 8080;
+const app = express();
+const listEndpoints = require("express-list-endpoints");
+
+// Root endpoint
+app.get("/", (req, res) => {
+  res.send(listEndpoints(app));
+});
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
+
+//Displays all awards
+app.get("/", (req, res) => {
+  res.json(data);
+});
+
+// Displays all the movies/shows
+app.get("/nominations", (req, res) => {
```

Comment:
Why `/nominations`? Wouldn't it be better to use `/shows`?

Reply:
I first went for Golden globes and then changed to Netflix. Seems I forgot to change the names lol 🙈